common/maplib/map.mjs

```diff
@@ -310,6 +332,25 @@ class _SingleGameMap {
     const tile = this.getCell(coord, layer);
     return this.graphicAsset.getTile(layer, tile);
   }
+  /**
+   * Get Spawn Point List.
+   * @param {int} movex , movey
+   * @return {list} spawn point - A list of the spawn point.
+   */
+  getSpawnPoint(movex, movey){
```

Comment:
This could be a more generic function, i.e. getCellsInStaticCellSet(), returning a list of MapCoord.